docs/src/components/landing/SignUp.js

```diff
@@ -0,0 +1,160 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import Alert from '@mui/material/Alert';
+import AlertTitle from '@mui/material/AlertTitle';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import CheckCircleRoundedIcon from '@mui/icons-material/CheckCircleRounded';
+import FormHelperText from '@mui/material/FormHelperText';
+import InputBase, { inputBaseClasses } from '@mui/material/InputBase';
+import KeyboardArrowRightRounded from '@mui/icons-material/KeyboardArrowRightRounded';
+import { styled } from '@mui/material/styles';
+import ROUTES from '../../route';
+
+const Form = styled('form')({});
+
+const SignUp = ({ sx }) => {
+  const [form, setForm] = React.useState({
+    email: '',
+    status: 'initial',
+  });
+  const handleSubmit = async (event) => {
+    event.preventDefault();
+    setForm((current) => ({ ...current, status: 'loading' }));
+    try {
+      await fetch(ROUTES.TOOLPAD_SIGN_UP_URL, {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/x-www-form-urlencoded',
+        },
+        mode: 'no-cors',
+        body: new URLSearchParams({
+          EMAIL: form.email,
+          email_address_check: '',
+          locale: 'en',
+        }),
+      });
+      setForm((current) => ({ ...current, status: 'sent' }));
+      window.ga('send', {
+        hitType: 'event',
+        eventCategory: 'ToolpadLanding',
+        eventAction: 'SignUp',
+        eventLabel: 'Sign Up',
+      });
+    } catch (error) {
+      setForm((current) => ({ ...current, status: 'failure' }));
+    }
+  };
+  if (form.status === 'sent') {
+    return (
+      <Alert
+        severity="success"
+        sx={{
+          maxWidth: { sm: 540 },
+          mx: { xs: 0, sm: 'auto' },
+          bgcolor: (theme) => (theme.palette.mode === 'dark' ? 'primaryDark.700' : 'success.50'),
+        }}
+        iconMapping={{
+          success: (
+            <CheckCircleRoundedIcon
+              fontSize="small"
+              sx={{
+                color: (theme) => (theme.palette.mode === 'dark' ? 'success.600' : 'success.700'),
+              }}
+            />
+          ),
+        }}
+      >
+        <AlertTitle sx={{ typography: 'body2', fontWeight: 700 }}>
+          Thanks! Check your email.
+        </AlertTitle>
+        You should get a <strong>confirmation email</strong> soon. Open it up and confirm your email
+        address so that we can keep you up to date.
+      </Alert>
+    );
+  }
+  return (
+    <Form onSubmit={handleSubmit} sx={sx}>
+      <Box
+        sx={{
+          display: 'flex',
+          gap: 0,
+          width: { xs: '100%', sm: 'auto' },
+          maxWidth: 320,
+        }}
+      >
+        <InputBase
```

Comment:
[WAVE](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) is reporting that we are missing a label

<img width="407" alt="Screenshot 2022-08-23 at 13 22 59" src="https://user-images.githubusercontent.com/3165635/186146038-4806a050-9e2b-48e7-b08a-fe81efff8b26.png">